cmd/podman/common/specgen.go

```diff
@@ -299,6 +299,14 @@ func FillOutSpecGen(s *specgen.SpecGenerator, c *ContainerCLIOpts, args []string
 		"container": "podman",
 	}
 
+	if defaultEnv, err := envLib.ParseSlice(containerConfig.Env()); err != nil {
```

Comment:
server side should call containerConfig.GetDefaultEnv()

Reply:
fixed